src/page-objects/StorefrontPages.ts

```diff
@@ -53,7 +55,7 @@ export const StorefrontPageObjects = {
     AccountAddresses,
     AccountPayment,
     Search,
-    SearchSuggest,
```

Comment:
We don't want to delete this SearchSuggest right?

Reply:
It is my mistake, fixed 🙈